ts/src/kucoin.ts

```diff
@@ -918,10 +919,10 @@ export default class kucoin extends Exchange {
         //    }
         //
         const data = this.safeValue (response, 'data');
-        return this.parseDepositWithdrawFee (data, currency);
+        return this.parseDepositWithdrawSingleFee (data, currency);
     }
 
-    parseDepositWithdrawFee (fee, currency = undefined) {
+    parseDepositWithdrawSingleFee (fee, currency = undefined) {
```

Comment:
@AlgoQ why do we need a different parser for the single version?

Reply:
The structures looks different, `fetchDepositWithdrawFee` also provides the chain:
```js
{
    "currency": "USDT",
    "limitBTCAmount": "1.00000000",
    "usedBTCAmount": "0.00000000",
    "remainAmount": "16548.072149",
    "availableAmount": "0",
    "withdrawMinFee": "25",
    "innerWithdrawMinFee": "0",
    "withdrawMinSize": "50",
    "isWithdrawEnabled": true,
    "precision": 6,
    "chain": "ERC20"
}
```

While in the structure of `fetchDepositWithdrawFees` the chain is not provided:
```js
{
    currency: 'ETH',
    name: 'ETH',
    fullName: 'Ethereum',
    precision: 8,
    confirms: 64,
    contractAddress: '',
    withdrawalMinSize: '0.01',
    withdrawalMinFee: '0.005',
    isWithdrawEnabled: true,
    isDepositEnabled: true,
    isMarginEnabled: true,
    isDebitEnabled: true
}
```

Which causes that when we use the same parse function this is the result:
```
{
  ETH: {
    info: {
      currency: 'ETH',
      name: 'ETH',
      fullName: 'Ethereum',
      precision: 8,
      confirms: 64,
      contractAddress: '',
      withdrawalMinSize: '0.01',
      withdrawalMinFee: '0.005',
      isWithdrawEnabled: true,
      isDepositEnabled: true,
      isMarginEnabled: true,
      isDebitEnabled: true
    },
    withdraw: { fee: undefined, percentage: undefined },
    deposit: { fee: undefined, percentage: undefined },
    networks: {
      undefined: {
        withdraw: { fee: undefined, percentage: undefined },
        deposit: { fee: undefined, percentage: undefined }
      }
    }
  },
  USDT: {
    info: {
      currency: 'USDT',
      name: 'USDT',
      fullName: 'Tether',
      precision: 8,
      confirms: 64,
      contractAddress: '0xdac17f958d2ee523a2206206994597c13d831ec7',
      withdrawalMinSize: '40',
      withdrawalMinFee: '25',
      isWithdrawEnabled: true,
      isDepositEnabled: true,
      isMarginEnabled: true,
      isDebitEnabled: true
    },
    withdraw: { fee: undefined, percentage: undefined },
    deposit: { fee: undefined, percentage: undefined },
    networks: {
      undefined: {
        withdraw: { fee: undefined, percentage: undefined },
        deposit: { fee: undefined, percentage: undefined }
      }
    }
  }
}
```

Reply:
@AlgoQ I know they're different but can't you detect one or the other and move this logic inside `parseDepositWithdrawFee`? 

For instance 
```Javascript
    parseDepositWithdrawFee (fee, currency = undefined) {
        if (singleFee) {
            return this.parseDepositWithdrawSingleFee (fee, currency);
        }
        // other logic
```
I'm insisting on this because if later on we decide to type `parseDepositWithdrawFee` in the base, `fetchDepositWithdrawFee` wouldn't get that type because uses a different parser